### PR TITLE
chore(build): ensure that importing from src is not used inside /src directory

### PR DESCRIPTION
**Pull request by tujoworker**
To prevent import path that are invalid in the final build we add:

- eslint rule
- postbuild test

Both are verified to work as expected:

Screenshot 2022-10-24 at 13 04 27

Screenshot 2022-10-24 at 11 18 29

### CODE REVIEW

**Comment by langz** [comment]
I guess it could/should must be necessary to import from a path containing the word src sometimes. I guess, as I see a lot of imports from paths containing the word src.

**Comment by tujoworker** [comment]
yeah, I also saw the lint check and made some changes [here](https://github.com/dnbexperience/eufemia/compare/a1ec60d5b87173c8cd4afef428fd019776c0f400..3af1aaf0edeed05755ca99a2b6edea2d2f59472e). I'm still not sure how to get listed all the committed files within a specific branch, because origin/main do not exists in a PR checkout on the CI.

**Review by langz — APPROVED** [review]
🚀

File: packages/dnb-eufemia/src/components/upload/UploadFileListCell.tsx [hunk 1]
```diff
@@ -6,7 +6,8 @@ import Button from '../button/Button'
 import Icon from '../../components/Icon'
 import FormStatus from '../../components/FormStatus'
 import ProgressIndicator from '../../components/progress-indicator'
-import P from '../../elements/P'
+// eslint-disable-next-line no-restricted-imports
+import P from '../../../src/elements/P'
```
Comment (langz) [inline packages/dnb-eufemia/src/components/upload/UploadFileListCell.tsx:10]: Forget to revert this 😂

Comment (tujoworker) [reply packages/dnb-eufemia/src/components/upload/UploadFileListCell.tsx:10]: impossible 😇

**Comment by tujoworker** [comment]
Finally, the [test does work](https://github.com/dnbexperience/eufemia/actions/runs/3319010700/jobs/5483628632). But for now, we only check the changes of the "current" commit. But that is almost as good as it gets. Added comments to the file where the command is run.

File: packages/dnb-eufemia/scripts/tools/cliTools.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+        return resolve(stdout)
+      })
+    } catch (e) {
+      reject(e)
+    }
+  })
+}
+
+/**
+ * Returns a list of files that where changed in the current branch
```
Comment (langz) [inline packages/dnb-eufemia/scripts/tools/cliTools.js:22]:
```suggestion
 * Returns a list of files that were changed in the current branch
```

**Pull request merged**